satpy/readers/aapp_mhs_l1c.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2020, 2021 Pytroll developers
+
+# Author(s):
+
+#   Adam Dybbroe <Firstname.Lastname@smhi.se>
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Reader for the AAPP AMSU-B/MHS level-1c data.
+
+https://nwp-saf.eumetsat.int/site/download/documentation/aapp/NWPSAF-MF-UD-003_Formats_v8.0.pdf
+
+"""
+
+import logging
+
+import dask.array as da
+import numpy as np
+
+from satpy import CHUNK_SIZE
+from satpy.readers.aapp_l1b import AAPPL1BaseFileHandler, create_xarray
+
+logger = logging.getLogger(__name__)
+
+
+LINE_CHUNK = CHUNK_SIZE ** 2 // 90
+
+MHS_CHANNEL_NAMES = ['1', '2', '3', '4', '5']
+MHS_CHANNEL_NAMES_SET = ('1', '2', '3', '4', '5')
+
+PLATFORM_NAMES = {15: 'NOAA-15',
+                  16: 'NOAA-16',
+                  17: 'NOAA-17',
+                  18: 'NOAA-18',
+                  19: 'NOAA-19',
+                  1: 'Metop-B',
+                  2: 'Metop-A',
+                  3: 'Metop-C',
+                  4: 'Metop simulator'}
+
+MHS_PLATFORMS = ['Metop-A', 'Metop-B', 'Metop-C', 'NOAA-18', 'NOAA-19']
+
+MHS_ANGLES = ['sensor_zenith_angle', 'sensor_azimuth_angle',
```

Comment:
Should this be called `MHS_ANGLE_NAMES` for consistency?

Reply:
Yes, I will fix...

Reply:
Done